paasta_tools/run-paasta-api-playground.py

```diff
@@ -0,0 +1,48 @@
+import json
+import os
+
+from paasta_tools.cli.utils import pick_random_port
+
+
+def main():
+    print("-------------------------------------------------------")
+    print(
+        "Please run export PAASTA_SYSTEM_CONFIG_DIR=etc_paasta_playground to continue"
+    )
+    print(
+        "Please set environment variable PAASTA_TEST_CLUSTER to the cluster you want to use."
+    )
+    print("-------------------------------------------------------")
+    cluster = os.environ.get("PAASTA_TEST_CLUSTER", "kind-emanelsabban-k8s-test")
+    config_path = "etc_paasta_playground"
+
+    # find unused ports
+    port = pick_random_port("paasta-dev-api")
+
+    # Generate api endpoints
+    api_endpoints = {"api_endpoints": {cluster: f"http://localhost:{port}"}}
+    api_endpoints_path = os.path.join(os.getcwd(), config_path, "api_endpoints.json")
```

Comment:
might be good to also move away from getcwd here and use `__FILE__` or something so that this doesn't do weird things should someone call it with a different working dir than expected